etcdmain/config_test.go

```diff
@@ -15,11 +15,72 @@
 package etcdmain
 
 import (
+	"fmt"
+	"io/ioutil"
+	"log"
 	"net/url"
+	"os"
 	"reflect"
+	"strings"
 	"testing"
+
+	"github.com/ghodss/yaml"
 )
 
+func createCfgFileHelper(b []byte) (*os.File, error) {
+	// Create a temporary config file.
```

Comment:
Oh sorry ! I thought I had cleaned this file. Will remove all of the unnecessary comments.